lib/docs/src/checkbox.stories.tsx

```diff
@@ -1,27 +1,106 @@
-import { Meta } from "@storybook/react";
-import { Checkbox, DivPx } from "../../core/src";
+import { useState } from "react";
+import { DivPx, Checkbox } from "../../core/src";
+import { Utils } from "./utils";
 
 export default {
-	title: "Draft/Checkbox",
+	title: "Components/Checkbox",
 	component: Checkbox,
-} as Meta;
-
-export const Primary = (): JSX.Element => (
-	<div>
-		<Checkbox indeterminate>Checkbox</Checkbox>
-		<DivPx size={8} />
-		<Checkbox defaultChecked>Checkbox</Checkbox>
-		<DivPx size={8} />
-		<Checkbox>Checkbox</Checkbox>
-		<DivPx size={8} />
-		<Checkbox disabled indeterminate>
-			Checkbox
-		</Checkbox>
-		<DivPx size={8} />
-		<Checkbox disabled defaultChecked>
-			Checkbox
-		</Checkbox>
-		<DivPx size={8} />
-		<Checkbox disabled>Checkbox</Checkbox>
-	</div>
+	argTypes: {
+		children: Utils.arg(null),
+		checked: Utils.arg("boolean"),
+		indeterminate: Utils.arg("boolean"),
+		disabled: Utils.arg("boolean"),
+		setChecked: Utils.arg(null),
+		defaultChecked: Utils.arg(null),
+		forwardedRef: Utils.arg(null),
+	},
+};
+
+interface Props {
+	indeterminate: boolean;
+	disabled: boolean;
+	checked: boolean;
+}
+
+export const Primary = (props: Props): JSX.Element => {
+	return (
+		<>
+			<Checkbox
+				defaultChecked
+				checked={props.checked}
+				disabled={props.disabled}
+				indeterminate={props.indeterminate}
+			>
+				Checkbox
+			</Checkbox>
+			<DivPx size={8} />
+		</>
+	);
+};
+
+Utils.fixPrimary(Primary);
+
```

Comment:
Also there should be a "Basic" story where you explain the most simple, best practice usage of the component